src/meta_data.rs

```diff
@@ -34,14 +35,15 @@ pub struct DatenLordNode {
     pub max_available_space_bytes: i64,
     /// Max volumes per node
     pub max_volumes_per_node: i32,
-    // /// Node IP
-    // pub ip_address: String, // TODO: add node IP address
+    /// Node IP
+    pub ip_address: IpAddr,
 }
 
 impl DatenLordNode {
     /// Create `DatenLordNode`
     pub const fn new(
         node_id: String,
+        ip_address: IpAddr,
```

Comment:
it's best to discover IP when create `DatenLordNode`, instead of input its IP

Reply:
Based on discussion, will keep this and ip address will be passed from outside.